airbyte-integrations/connectors/source-postgres/src/test/java/io/airbyte/integrations/source/postgres/PostgresSpecTest.java

```diff
@@ -157,4 +157,39 @@ void testLsnCommitBehaviourProperty(final String commitBehaviour) {
     assertTrue(validator.test(schema, config));
   }
 
+  @Test
+  void testLsnCommitBehaviourPropertyWithWrongValue() {
+    final JsonNode replicationMethod = Jsons.jsonNode(ImmutableMap.builder()
+            .put(“method”, “CDC”)
```

Comment:
@dalo390 you're not using the correct `"` character here. Please it.

Reply:
Thank you for the catch, I have fixed the characters @marcosmarxm